### PR TITLE
Remove high_res_img() helper (Fixes #13227)

### LINKED ISSUE #13227: Deprecate the `high_res_img()` helper
## Description

There are a lot of instances of the older `high_res_img()` helper that can/should be switched over to the newer `resp_img()` helper.

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Removes all usage of `high_res_img()` and replaces it with `resp_img()`, so we finally have just a single image helper that can do everything :)

## Significant changes and points to review

Did i miss any stray occurrences of `high_res_img()`

**Note:** For the VPN platform pages, you may need to switch the template that gets shown in the view in order to see the older pages: https://github.com/mozilla/bedrock/blob/main/bedrock/products/views.py#L216-L217

## Issue / Bugzilla link

#13227

## Testing

- [ ] http://localhost:8000/en-US/firefox/browsers/mobile/android/
- [ ] http://localhost:8000/en-US/firefox/browsers/chromebook/
- [ ] http://localhost:8000/en-US/firefox/browsers/compare/brave/
- [ ] http://localhost:8000/en-US/firefox/browsers/compare/chrome/
- [ ] http://localhost:8000/en-US/firefox/browsers/compare/edge/
- [ ] http://localhost:8000/en-US/firefox/browsers/compare/ie/
- [ ] http://localhost:8000/en-US/firefox/browsers/compare/opera/
- [ ] http://localhost:8000/en-US/firefox/browsers/compare/safari/
- [ ] http://localhost:8000/en-US/firefox/browsers/
- [ ] http://localhost:8000/en-US/firefox/browsers/mobile/
- [ ] http://localhost:8000/en-US/firefox/developer/
- [ ] http://localhost:8000/en-US/firefox/features/adblocker/
- [ ] http://localhost:8000/en-US/firefox/flashback/
- [ ] http://localhost:8000/en-US/firefox/more/misinformation/
- [ ] http://localhost:8000/en-US/firefox/new/
- [ ] http://localhost:8000/en-US/firefox/products/
- [ ] http://localhost:8000/en-US/foundation/annualreport/2011/
- [ ] http://localhost:8000/en-US/foundation/trademarks/list/
- [ ] http://localhost:8000/en-US/about/legal/
- [ ] http://localhost:8000/en-US/about/
- [ ] http://localhost:8000/en-US/about/leadership/
- [ ] http://localhost:8000/en-US/about/leadership/senior-leadership/
- [ ] http://localhost:8000/en-US/about/leadership/mozilla-foundation/
- [ ] http://localhost:8000/en-US/about/leadership/reps-council/
- [ ] http://localhost:8000/en-US/about/leadership/boards-of-directors/
- [ ] http://localhost:8000/en-US/about/manifesto/
- [ ] http://localhost:8000/en-US/about/policy/lean-data/
- [ ] http://localhost:8000/en-US/about/policy/lean-data/stay-lean/
- [ ] http://localhost:8000/en-US/about/policy/lean-data/build-security/
- [ ] http://localhost:8000/en-US/about/policy/lean-data/engage-users/
- [ ] http://localhost:8000/en-US/contact/spaces/san-francisco/
- [ ] http://localhost:8000/en-US/contact/spaces/beijing/
- [ ] http://localhost:8000/en-US/contact/spaces/berlin/
- [ ] http://localhost:8000/en-US/contact/spaces/paris/
- [ ] http://localhost:8000/en-US/contact/spaces/toronto/
- [ ] http://localhost:8000/en-US/contribute/
- [ ] http://localhost:8000/en-US/
- [ ] http://localhost:8000/de/
- [ ] http://localhost:8000/fr/
- [ ] http://localhost:8000/es-ES/
- [ ] http://localhost:8000/en-US/moss/
- [ ] http://localhost:8000/de/products/vpn/
- [ ] http://localhost:8000/en-US/products/vpn/mobile/android/
- [ ] http://localhost:8000/en-US/products/vpn/mobile/ios/
- [ ] http://localhost:8000/en-US/products/vpn/mobile/
- [ ] http://localhost:8000/en-US/products/vpn/desktop/
- [ ] http://localhost:8000/en-US/products/vpn/desktop/windows/
- [ ] http://localhost:8000/en-US/products/vpn/desktop/mac/
- [ ] http://localhost:8000/en-US/products/vpn/desktop/linux/
- [ ] http://localhost:8000/en-US/products/vpn/pricing/
- [ ] http://localhost:8000/de/products/vpn/pricing/?geo=cn

In Pocket mode:

- [ ] http://localhost:8000/en/jobs/

### CODE REVIEW

**Comment by alexgibson** [comment]
Successful test run: http://localhost:8000/en/jobs/

File: bedrock/mozorg/templates/mozorg/about/leadership/reps.html
Comment (reemhamz) [inline bedrock/mozorg/templates/mozorg/about/leadership/reps.html:1]: **Question:** I wonder if we need to keep high-res images in this file, seeing as how the folks listed here don't have modals that open with larger images needed (as opposed to the other leadership pages). Their images are going to be quite small on any screen as it is, do you think it's a safe bet to remove high-res images here altogether?

**Review by reemhamz — APPROVED** [review]
Great going with this! It was a big one.
I just have one question that I think is worth visiting, but other than that, r+(wc)! 🧠

File: bedrock/mozorg/templates/mozorg/about/leadership/reps.html
Comment (alexgibson) [reply bedrock/mozorg/templates/mozorg/about/leadership/reps.html:1]: I took a look and the images are rendered at 160x160px on screen, so it seems appropriate still to serve 320x320px images to high res screens. If we removed them then the images would probably look a bit blurry.

We could instead drop the low res images altogether, however then we'd be serving images twice the size we'd need to people on low res screens. I'm not sure what % of visitors are still on low res screens these days, but that's probably something that's worth exploring at a site level some day. For now, i think we're probably fine to keep things as they are. It's a good question though!

**Comment by alexgibson** [comment]
Thanks for the review @reemhamz!

**Pull request merged**